.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -5,36 +5,55 @@ about: Create a report to help us improve
 
 <!-- Before you open a new issue, search through the existing issues to see if others have had the same problem.
 
+DO NOT DELETE ANYTHING FROM THIS TEMPLATE.
+
 Issues not containing the minimum requirements will be closed:
 
-- Issues without a description (using the header is not good enough) will be closed.
-- Issues without debug logging will be closed.
-- Issues without configuration will be closed
+To help us to resolve this issue please include the following information:
+- Describe the issue in detail.
+- Provide steps to reproduce this issue.
+- Add version of the `google_home` integration.
+- Add version of `glocaltokens` package.
+- Attach logs. The more the better. Enable debug logs for the integration.
 
 -->
 
-## Version of the custom_component
+## Describe the bug
+
+<!-- A clear and concise description of what the bug is. -->
+
+## Version of the integration
```

Comment:
Can we please mention GitHub releases (with a link) instead of 'const' file for versions?

Reply:
I am not sure i understand. Do you just want a link to [relaeses](https://github.com/leikoilja/ha-google-home/releases)?

Reply:
Yes yes, sorry, github doesn't allow to select the lines that are not added in this PR (Lines 27-29).

I was just proposing we say:

> If you are not using the newest version, download and try that before opening an issue
> If you are unsure about the version check the [github releases page](https://github.com/leikoilja/ha-google-home/releases).

Reply:
Will push a commit later 😄